src/modules/swap/hooks/useIsSwapEth.ts

```diff
@@ -0,0 +1,10 @@
+import { useDetectNativeToken } from './useDetectNativeToken'
+
+import { useIsWrapOrUnwrap } from '../../trade/hooks/useIsWrapOrUnwrap'
+
+export function useIsSwapEth(): boolean {
```

Comment:
New helper hook to identify whether this is a sell native token, re-used on both ethflow related hooks